File: Javascript/Basic Recursion/Problme2.js

function printNums(i,n){
    if(i>n){
      return 
    }
    console.log(i);
    printNums(i+1,n);
  }
  
  
  function main(){
    let n=10;
    printNums(1,n)
  }
  
  
  main()
  

File: C++/Basic Recursion/Problme2.cpp

// Print 1 to N using recursion


#include <iostream>
using namespace std;


void printNums(int i,int n){
  if(i>n){
    return;
  }
  std::cout << i << std::endl;
  
  printNums(i+1,n);
}

int main() 
{
    int n=10;
    printNums(1,n);
    return 0;
}